arthur_bench/server/run_server.py

```diff
@@ -152,6 +163,7 @@ def test_run_results(
     run_id: uuid.UUID,
     page: int = 1,
     page_size: int = 5,
+    sort: Annotated[Optional[TestCaseSortEnum], Query()] = TestCaseSortEnum.ORDER_ASC,
```

Comment:
nit: for these I think its better practice to shove all the information / validation for these params into query(), otherwise its not really used. So this and others would be `sort: Annotated[Optional[TestCaseSortEnum], Query(default=TestCaseSortEnum.ORDER_ASC)]`

Reply:
nevermind, this is the correct pattern: https://fastapi.tiangolo.com/tutorial/query-params-str-validations/#query-as-the-default-value-or-in-annotated